pkg/api/store/pod/store.go

```diff
@@ -62,12 +59,9 @@ func streamTransform(context *types.APIContext, data chan map[string]interface{}
 	result := make(chan map[string]interface{})
 	go func() {
 		for item := range data {
-			typeName := definition.GetType(item)
-			if strings.Contains(typeName, "replica") || strings.Contains(typeName, "deployment") {
```

Comment:
@ibuildthecloud when debugged the old code, "typeName" here comes as "pod", so the mapping never get refreshed.